cdap-ui/app/cdap/components/AbstractWidget/HierarchyWidget/dataHandler.ts

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IStageSchema } from 'components/AbstractWidget';
+import uuidV4 from 'uuid/v4';
+
+interface IField {
+  name: string;
+  type: any;
```

Comment:
Can we narrow down on the type here? `any` type in an interface just loses the advantage of typescript.